data_collection/gazette/spiders/sp_sorocaba.py

```diff
@@ -0,0 +1,100 @@
+import re
+from datetime import date, datetime
+
+from scrapy import Request
+from dateparser import parse
+from dateutil.rrule import MONTHLY, rrule
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpSorocabaSpider(BaseGazetteSpider):
+    name = "sp_sorocaba"
+    TERRITORY_ID = "3552205"
+
+    start_date = date(2005, 1, 7)
+
+    gazettes_list_url = "http://noticias.sorocaba.sp.gov.br/jornal/?ano={}"
+
+    MonthDict = {

```

Comment:
**suggestion**: Generally constants names are all in uppercase. It is not usual to have the type of the variable on its name too. `MONTHS` is a better name for this variable.